library/Cube/Web/IcingaDbActionLinks.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// Icinga Web 2 Cube Module | (c) 2016 Icinga GmbH | GPLv2
+
+namespace Icinga\Module\Cube\Web;
+
+use Exception;
+use Icinga\Application\Hook;
+use Icinga\Module\Cube\Cube;
+use Icinga\Module\Cube\Hook\IcingaDbActionsHook;
+use Icinga\Module\Cube\IcingaDb\IcingaDbCube;
+use ipl\Html\HtmlDocument;
+use ipl\Html\HtmlElement;
+use ipl\Web\Widget\Link;
+
+/**
+ * ActionLink
+ *
+ * ActionsHook implementations return instances of this class
+ *
+ * @package Icinga\Module\Cube\Web
+ */
+class IcingaDbActionLinks
```

Comment:
This class will be obsolete once you've changed `IcingaDbActionsHook::getActionLinks()`.

* `add()` and `getLinks()` are unused then.
* `render()` is already now unused.
* `addErrorItem()` is not that specific that I can't imagine its functionality being a part of `renderAll` instead.
* `renderAll()` then is the only remaining method. It should be a part of `IcingaDbActionsHook`, as I already requested previously.